tests/unit/trace/exporters/test_zipkin_exporter.py

```diff
@@ -202,6 +205,178 @@ def test_translate_to_zipkin_span_kind_none(self):
                 'localEndpoint': local_endpoint_ipv6,
                 'tags': {'test_key': 'False', 'test_key2': 'raw_value'},
                 'kind': 'SERVER',
+                'annotations': [],
+            },
+        ]
+
+        # Test ipv4 local endpoint
+        exporter_ipv4 = zipkin_exporter.ZipkinExporter(
+            service_name='my_service', ipv4=ipv4)
+        zipkin_spans_ipv4 = exporter_ipv4.translate_to_zipkin(
+            span_datas=spans_ipv4)
+
+        self.assertEqual(zipkin_spans_ipv4, expected_zipkin_spans_ipv4)
+
+        # Test ipv6 local endpoint
+        exporter_ipv6 = zipkin_exporter.ZipkinExporter(
+            service_name='my_service', ipv6=ipv6)
+        zipkin_spans_ipv6 = exporter_ipv6.translate_to_zipkin(
+            span_datas=spans_ipv6)
+
+        self.assertEqual(zipkin_spans_ipv6, expected_zipkin_spans_ipv6)
+
+    def test_translate_to_zipkin_with_annotations(self):
+        trace_id = '6e0c63257de34c92bf9efcd03927272e'
+
+        annotation_attributes = {
+            'annotation_bool': True,
+            'annotation_string': 'annotation_test',
+            'key_float': .3
+        }
+
+        import datetime
```

Comment:
Please put imports at the top and not unexpected in the body of a test.

Reply:
Done